### PR TITLE
SUS-2370 Remove AutomaticWikiAdopt

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: extensions/wikia/WikiFactory/WikiFactory.php [hunk 1]
```diff
@@ -68,9 +68,9 @@ class WikiFactory {
 	const FLAG_FREE_WIKI_URL         = 8; // removes wiki database and WikiFactory settings
 	const FLAG_HIDE_DB_IMAGES        = 16;  // images and DB dumps will be hidden on s3
 	const FLAG_REDIRECT              = 32;  // this wiki is a redirect - do not remove
-	const FLAG_ADOPTABLE             = 64;  //used by AutomaticWikiAdoption
-	const FLAG_ADOPT_MAIL_FIRST      = 128; //used by AutomaticWikiAdoption
-	const FLAG_ADOPT_MAIL_SECOND     = 256; //used by AutomaticWikiAdoption
+	const FLAG_ADOPTABLE             = 64;  //used by AutomaticWikiAdoption (removed)
+	const FLAG_ADOPT_MAIL_FIRST      = 128; //used by AutomaticWikiAdoption (removed)
+	const FLAG_ADOPT_MAIL_SECOND     = 256; //used by AutomaticWikiAdoption (removed)
```
Comment (v3rth) [inline extensions/wikia/WikiFactory/WikiFactory.php]: Should I leave these flags?

File: maintenance/wikia/AutomaticWikiAdoptionGatherData.php [hunk 2]
```diff
@@ -1,32 +0,0 @@
...
- * @license http://www.gnu.org/copyleft/gpl.html GNU General Public License 2.0 or later
- * @package MediaWiki
- * @subpackage Maintanance
- *
- */
-
-require_once( dirname(__FILE__) . '/../commandLine.inc' );
-
-if (isset($options['help'])) {
-	die( "Usage: php AutomaticWikiAdoptionGatherData.php [--quiet] [--maxwiki=12345]
```
Comment (v3rth) [inline maintenance/wikia/AutomaticWikiAdoptionGatherData.php:20]: How to make sure that this command is no longer used?

File: extensions/wikia/WikiFactory/WikiFactory.php (hunk 1 above)
Comment (mszabo-wikia) [reply extensions/wikia/WikiFactory/WikiFactory.php]: imho kill them. @macbre ?

File: maintenance/wikia/AutomaticWikiAdoptionGatherData.php (hunk 2 above)
Comment (mszabo-wikia) [reply maintenance/wikia/AutomaticWikiAdoptionGatherData.php:20]: if it's not configured to run in chef-repo and not referred to here in app either, then it's essentially unused :)

File: extensions/wikia/WikiFactory/WikiFactory.php (hunk 1 above)
Comment (macbre) [reply extensions/wikia/WikiFactory/WikiFactory.php]: Given they're no longer used, let's remove them.

Comment (v3rth) [reply extensions/wikia/WikiFactory/WikiFactory.php]: Ok, thanks! Removed

File: maintenance/wikia/AutomaticWikiAdoptionGatherData.php (hunk 2 above)
Comment (v3rth) [reply maintenance/wikia/AutomaticWikiAdoptionGatherData.php:20]: I couldn't find anything, so for me it looks good, but please double check it while CR

**Review by mszabo-wikia — APPROVED** [review]

File: maintenance/wikia/AutomaticWikiAdoptionGatherData.php (hunk 2 above)
Comment (macbre) [reply maintenance/wikia/AutomaticWikiAdoptionGatherData.php:20]: https://github.com/Wikia/chef-repo/commit/1e673c3eb59479f7c4ecbe7eb952f29371505b43 - `AutomaticWikiAdoptionGatherData` was added as a cronjob back in 2010

And then removed in 2014 in https://github.com/Wikia/chef-repo/commit/7058d46af0c794eef35de1e9ffb602a7bd883a1e

**Review by macbre — APPROVED** [review]

**Comment by macbre** [comment]
I'd add a database cleanup step after we merge and deploy this change:

```
-		$dbr = wfGetDB(DB_SLAVE, array(), $wgExternalDatawareDB);
-		$dismissed = $dbr->selectField(
-			'user_flags',
-			'data',
-			array('city_id' => $wgCityId, 'user_id' => $user->getID(), 'type' => self::USER_FLAGS_AUTOMATIC_WIKI_ADOPTION),
-			__METHOD__
-		);
```

`USER_FLAGS_AUTOMATIC_WIKI_ADOPTION = 1`

**Pull request merged**